Introductory/README.md

```diff
@@ -1,6 +1,6 @@
 ### Introduction to VANTIQ Development
 The Engine Monitor tutorial walks through developing and testing VANTIQ Apps and Clients to monitor the speed and temperature of an engine. 
-An app is used to process the stream of sensor readings and detect anomolous conditions in the engine, and a client is used
+An app is used to process the stream of sensor readings and detect anomalous conditions in the engine, and a client is used
 to visualize the data and display any alerts as they occur.
 
-For more information on the Engine Monitor Tutorial, check out the documentation [here](/docs/system/tutorials/tutorial/index.html).
+For more information on the Engine Monitor Tutorial, check out the documentation [here](../../docs/locale/en/md/dev/tutorials/tutorial.md).
```

Comment:
I don't think this change is right.  (And I made the same mistake here last release.)
The URL has to be relative to the UI url, not the github url, because this URL is shown in the Project -> Import dialog and we want that to go to the docs in the product, not github.
so revert this change, please.

@JakeSchmitz, @mjswan, could we also improve these links to open up in a new tab when clicked in that dialog?  Currently, they use the same tab as the UI and replace the UI...